Turner.Infrastructure.Mediator.Tests/Fakes/FakeDbContext.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Diagnostics;
```

Comment:
Is this using required?

Reply:
Yes, upgraded the package and the new one changes where `InMemoryEventId` is housed.
![image](https://user-images.githubusercontent.com/4959971/32620914-3e8318e2-c544-11e7-8af8-69562825ee2f.png)

Reply:
Ah okay